week0004/thainguyentran/CSharp/week0004.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace DVRK
+{
+    class Program
+    {
+        public static Dictionary<string, string> guestList;
+        public static string[] event1;
```

Comment:
Remove event1 & event2. You can have them in Main. Also I would prefer you to have a separate class for processing and the Program class will use it.